chef/cookbooks/ceilometer/recipes/common.rb

```diff
@@ -66,7 +66,7 @@
   bind_host = admin_address
   bind_port = node[:ceilometer][:ha][:ports][:api]
 else
-  bind_host = "0.0.0.0"
+  bind_host = node[:ceilometer][:api][:host]
```

Comment:
Any reason for that change? (I'm not against it, just sounds irrelevant, maybe split it in a different commit but keep it in this PR)